web-controller/src/main/java/com/oasis/webcontroller/LoginController.java

```diff
@@ -0,0 +1,41 @@
+package com.oasis.webcontroller;
+
+import com.oasis.model.EmployeeModel;
+import com.oasis.service.LoginService;
+import com.oasis.webmodel.request.LoginRequestModel;
+import com.oasis.webmodel.response.BaseResponse;
+import com.oasis.webmodel.response.LoginResponseModelFAIL;
+import com.oasis.webmodel.response.LoginResponseModelOK;
+import com.oasis.webmodel.response.ResponseStatuses;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class LoginController {
+    @Autowired
+    private LoginService loginService;
+
+    @RequestMapping(value = MappingValues.LOGIN_MAPPING_VALUE, method = RequestMethod.POST, produces = "application/json", consumes = "application/json")
```

Comment:
change "application/json" to MediaType.APPLICATION_JSON_VALUE

Reply:
Commit ID: 2029415